### ISSUE TITLE
can't override base metals /modern / nether / end materials for TIC

### ISSUE THREAD

**Issue by cosmo3001** [tyra314/ToolProgression#14]
hi i love the mod, don't know what to do without it.
i am trying to make a modpack, with focus on tool progression.
for that i have base metals / modern / nether / end installed in the pack. When i override the material in material override it don't change anything in TIC, they remain as listed in material configs.
Is there any other way i can do it so that TiC will recognize the change made to the material?

**Comment by tyra314** [comment]
TL;DR Fixed one bug, should work now. Please be adviced, you need to specify overwrites for all different names of the same material.

For this issue to happen, it needs a number of things to line up. Firstly, the overwrite itself works, if you could have known how to make it. The problem here is that ToolProgression had a bug [1], which prevented TiC Materials to show up in the `materials.cfg`. Unfortunately, the * metals mods seem to register materials twice. One with a prefix for vanilla-like tools and one without prefix for TiC. For instance, there is `basemetals_bismuth` and `bismuth`. Because for my mod these are two different materials, overwriting one, doesn't change the other. As you only saw the one for vanilla in `materials.cfg`, it's fair game to only specify this one. Easy fix right now, add another overwrite for the missing TiC materials.


[1] This bug is fixed in the version [v1.4.3](https://minecraft.curseforge.com/projects/tool-progression/files/2505339)

**Comment by cosmo3001** [comment]
oh damn i did not give enough info.
im working with mc 1.11.2 would it be possible for you to add the fix there as well?

**Comment by tyra314** [comment]
Well, it's still pretty much limited regarding the used versions, but I'll have a look. No promise though, if it ain't an easy fix or a crash, I'm not going to put much effort into 1.11 anymore.

**Comment by cosmo3001** [comment]
thank you soooo much. and yeah if it's hell don't bother

**Comment by tyra314** [comment]
The upcoming release should also fix this for 1.11.2.

https://minecraft.curseforge.com/projects/tool-progression/files/2505872

**Comment by cosmo3001** [comment]
Thank you so much :)
